components/usage/pkg/db/workspace_instance.go

```diff
@@ -176,6 +177,11 @@ func (a AttributionID) Values() (entity string, identifier string) {
 	return tokens[0], tokens[1]
 }
 
+func (a AttributionID) IsUser() bool {
+	entity, _ := a.Values()
+	return entity == "user"
+}
```

Comment:
[personal] I generally prefer not to introduce Binary methods on an object which can have more than a binary number of outcomes. This forces the caller to use 
```
switch a.Kind {
  case User:
    ...
  case Team:
    ...
  default:
    ...
}
```
This tends to be more resilient over time as you don't make binary choices anymore, but are forced to acknowledge existence of other values.